### PR TITLE
For EmberTimes No. 92: Contributor Interview

### PR DESCRIPTION
**Pull request by jayjayjpg**
## Sources

Screenshot 2019-04-02 at 22 23 07

### CODE REVIEW

File: source/2019-04-05-the-ember-times-issue-92.md [hunk 1]
```diff
@@ -29,56 +33,63 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
...
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## ["I contribute to Ember" with Chris Garrett 💬](https://discuss.emberjs.com/t/i-contribute-to-ember-with-chris-garrett/16395)
+
+<div class="float-right padded portrait-frame">
+  <img alt="Chris Garrett" title="Chris Garrett - Contributor to Ember" src="/images/blog/emberjstimes/pzuraq.jpg" />
+</div>
+
+In the 9th edition of the contributor interview series, community member **Chris Garrett**, also known as [@pzuraq](https://github.com/pzuraq) talks about his story getting involved with working on **native classes** and **decorators** in Ember and why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```
Comment (ijlee2) [inline source/2019-04-05-the-ember-times-issue-92.md]: @jessica-jordan I think your section looks good overall (it's interesting to see how we use Markdown!). This sentence may be long and could be divided into two? How about,

In the 9th edition of the contributor interview series, community member **Chris Garrett**, also known as [@pzuraq](https://github.com/pzuraq), shares his story of getting involved with **native classes** and **decorators** in Ember. Chris also explains why everyone in the community is eager to help **make your contribution** a real **success** ✨

Cheers,

Comment (jaredgalanis) [inline source/2019-04-05-the-ember-times-issue-92.md]: Looks ✨ !!!

I would consider adding a comma after the reference to the pzuraq handle and link or moving the comma trailing Chris Garrett to after the handle and link and putting that and the `also known as` in parens.

**Review by amyrlam — APPROVED** [review]
Agree with Jared's suggestion, but besides that looks good! 💯

**Review by ijlee2 — APPROVED** [review]

File: source/2019-04-05-the-ember-times-issue-92.md (hunk 1 above)
Comment (jayjayjpg) [reply source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
In the 9th edition of the contributor interview series, community member **Chris Garrett** (also known as [@pzuraq](https://github.com/pzuraq)) talks about his story getting involved with working on **native classes** and **decorators** in Ember and why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```

Comment (jayjayjpg) [reply source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
In the 9th edition of the contributor interview series, community member **Chris Garrett**, also known as [@pzuraq](https://github.com/pzuraq), talks about his story getting involved with **native classes** and **decorators** in Ember. Chris also explains why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```

File: source/2019-04-05-the-ember-times-issue-92.md [hunk 2]
```diff
@@ -29,56 +33,63 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
...
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## ["I contribute to Ember" with Chris Garrett 💬](https://discuss.emberjs.com/t/i-contribute-to-ember-with-chris-garrett/16395)
+
+<div class="float-right padded portrait-frame">
+  <img alt="Chris Garrett" title="Chris Garrett - Contributor to Ember" src="/images/blog/emberjstimes/pzuraq.jpg" />
+</div>
+
+In the 9th edition of the contributor interview series, community member **Chris Garrett** (also known as [@pzuraq](https://github.com/pzuraq)) talks about his story getting involved with working on **native classes** and **decorators** in Ember and why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```
Comment (jayjayjpg) [inline source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
In the 9th edition of the contributor interview series, community member **Chris Garrett**, also known as [@pzuraq](https://github.com/pzuraq), talks about his story getting involved with **native classes** and **decorators** in Ember. Chris also explains why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```

**Comment by jayjayjpg** [comment]
Awesome, thank you for the swift read through and review @jaredgalanis, @ijlee2 & @amyrlam! :sparkles:

**Pull request merged**